variants/STM32MP157_DK/README.md

```diff
@@ -13,49 +13,49 @@ Because every OS may have different software configurations (especially Device T
 * [STM32 MPU OpenSTLinux Distribution]
 * [Balena OS]
 
-Note that the first two OSes should select `stm32mp157c-dk2-m4-examples-sdcard` boot mode (or device tree configuration) on boot time. `stm32mp157c-dk2-sdcard` boot mode is also known to work but it is not guaranteed. See [ST Wiki page on boot mode] for more detail. If you are using it for your own OS, see the [Linux Device Tree considerations](#Linux-Device-Tree-considerations) section.
+__Note that the first two OSes should select `stm32mp157c-dk2-m4-examples-sdcard` boot mode (or device tree configuration) on boot time.__ `stm32mp157c-dk2-sdcard` boot mode is also known to work but it is not guaranteed. See [ST Wiki page on boot mode] for more detail. If you are using it for your own OS, see the [Linux Device Tree considerations](#Linux-Device-Tree-considerations) section.
```

Comment:
Because our generate script is quite generic we can add this note?
```suggestion
Other distributions not listed here may run Arduino but they are not tested. __Note that the first two OSes should select `stm32mp157c-dk2-m4-examples-sdcard` boot mode (or device tree configuration) on boot time.__ `stm32mp157c-dk2-sdcard` boot mode is also known to work but it is not guaranteed. See [ST Wiki page on boot mode] for more detail. If you are using it for your own OS, see the [Linux Device Tree considerations](#Linux-Device-Tree-considerations) section.
```

Reply:
Note added

Reply:
Great, thanks for updating! :)

Reply:
Welcome 😉 
Thanks too for your review. 👍